docs/DeveloperGuide.md

```diff
@@ -198,18 +198,35 @@ The activity diagram below shows the execution of the above example:
   * Pros: `Participant` has less dependence on `Contact`. Better flexibility as each `Participant` no longer needs to be a `Contact`.
```

Comment:
I think under the `Activity Diagram` section above you might want to link the reader to the relevant section if they are interested in finding out more about the `Create Participant` part of the diagram.